app/processing/delta/calculate-delta.js

```diff
@@ -1,4 +1,5 @@
 const { allocateToLedgers } = require('./assign-ledger')
+const { getDefaultAgreementNumber } = require('./get-default-agreement-number')
```

Comment:
I can only make a comment here, but put these definitions in the order they're used below?